packages/components/src/Tooltip.tsx

```diff
@@ -25,7 +25,7 @@ export const Tooltip = ({
         position === "top" &&
           "top-0 -translate-y-[calc(100%-10px)]  group-hover/tooltip:-translate-y-full",
         position === "bottom" &&
-          "top-auto bottom-0 translate-y-[calc(100%-10px)] group-hover/tooltip:translate-y-[calc(100%+0.25em)]",
+          "top-auto bottom-0 translate-y-[calc(100%-10px)] group-hover/tooltip:translate-y-full",
```

Comment:
I had to change this, otherwise tooltip can disappear because of the space between the icon and the text.